client/components/Slide/Components/MultiButtonResponse/Editor.jsx

```diff
@@ -116,22 +128,54 @@ class MultiButtonResponseEditor extends React.Component {
         this.setState({ buttons }, this.updateState);
     }
 
+    toggleOptional(event, { index }) {
+        const { activeIndex } = this.state;
+        const newIndex = activeIndex === index ? -1 : index;
+
+        this.setState({ activeIndex: newIndex });
+    }
+
     render() {
-        const { buttons, prompt, required } = this.state;
+        const { scenarioId } = this.props;
+        const { activeIndex, buttons, prompt, recallId, required } = this.state;
         const {
             onAddButton,
             onChangeButtonDetail,
             onChangeButtonOrder,
+            onRecallChange,
             onChangePrompt,
             onDeleteButton,
             onFocusButtonDetail,
             onRequirementChange,
+            toggleOptional,
             updateState
         } = this;
+
         return (
             <Form>
+                <Accordion>
```

Comment:
Non-blocking: it would be nice to have this according dropdown menu as its own component since it appears in a couple places now.

Reply:
I agree, but would like to defer to a follow up task, time permitting :)